fawltydeps/packages.py

```diff
@@ -345,6 +347,15 @@ def find_package_dirs(cls, path: Path) -> Iterator[Path]:
             if found:
                 return
 
+        # Check for packages on Windows
+        if platform.system() == "Windows":
+            for site_packages in path.glob(os.path.join("Lib", "site-packages")):
+                if site_packages.is_dir():
+                    yield site_packages
+                    found = True
+            if found:
+                return
+
```

Comment:
There are [many different ways](https://stackoverflow.com/questions/1325581/how-do-i-check-if-im-running-on-windows-in-python) of checking if we're running on Windows. I think I'm partial to using [`sys.platform`](https://docs.python.org/3/library/sys.html#sys.platform) rather than `platform.system()` for the following reasons:
- We're already importing `sys` ~everywhere, so fewer imports are needed
- `sys.platform` differentiates between `"win32"` and `"cygwin"` which is useful here (since Unix-style paths are expected in a cygwin environment).

Also, I notice there are no wildcards (`?`, `*`) passed to `path.glob()`. Maybe we don't need to call `glob` at all?

Finally, I'd like to locate the Windows site-packages check somewhat closer to the corresponding Unix-style check.

Maybe something like this (starting around line 332)?

```python
        found = False

        if sys.platform.startswith("win"):  # Windows
            site_packages = path / "Lib" / "site-packages"
            if site_packages.is_dir():
                yield site_packages
                found = True
        else:  # Assume POSIX
            if (path / "bin/python").is_file():
                for site_packages in path.glob("lib/python?.*/site-packages"):
                    if site_packages.is_dir():
                        yield site_packages
                        found = True
        if found:
            return
```

Reply:
BTW, I notice further below that a Windows virtualenv contains a
```
python_path = venv_path / "Scripts" / "python.exe"
```
Do we want to check for this, just like we're checking `if (path / "bin/python").is_file():` in the POSIX case?

Reply:
Fixed in https://github.com/tweag/FawltyDeps/commit/a5fb06736edf091ec1c657926886d310dda005d2.